presto-base-jdbc/src/test/java/com/facebook/presto/plugin/jdbc/TestJdbcDistributedQueries.java

```diff
@@ -30,4 +30,10 @@ public TestJdbcDistributedQueries()
     public void testLargeIn()
     {
     }
+
+    @Override
+    public void testShowTablesLikeWithEscape()
+    {
+        // Tpch connector currently does not support create table
```

Comment:
Thanks! Next time it would be sufficient to write `the connector...` -- then it's easier to copy-paste :)